.travis.yml

```diff
@@ -27,7 +29,7 @@ branches:
 before_install:
   - if test "$TRAVIS_OS_NAME" == "osx"; then brew update; brew install openssl; ln -s /usr/local/opt/openssl/lib/libcrypto.1.0.0.dylib /usr/local/lib/; ln -s /usr/local/opt/openssl/lib/libssl.1.0.0.dylib /usr/local/lib/; fi
 script:
-  - ./build.sh verify
+  - ./build.sh --quiet verify
```

Comment:
This change reduces informational messages that seem to be of interest primarily to the `dotnet` CLI team.

Errors and warnings are still logged. In fact, errors and warnings are easier to find because there's less noise around them. We have not had to turn `--quiet` off to debug Travis failures in repos where this is already used.